config/jobs/kubernetes-sigs/service-apis/service-apis-config.yaml

```diff
@@ -23,3 +23,26 @@ presubmits:
         # docker-in-docker needs privileged mode.
         securityContext:
           privileged: true
+  - name: pull-service-apis-test
+    annotations:
+      testgrid-dashboards: sig-network-service-apis
+      testgrid-tab-name: test
+    labels:
+      preset-kind-volume-mounts: "true"
+      preset-dind-enabled: "true"
+    decorate: true
+    path_alias: sigs.k8s.io/service-apis
+    always_run: true
+    skip_report: false
```

Comment:
should you start with `optional: true` to avoid that if something fails it blocks other PR to merge?

Reply:
Can you link where the optional field is detailed ? I can't figure out how some of these flags work.

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#requiring-jobs-for-auto-merge-through-tide
it means that it doesn't block the merge if it fails, however, if you are sure that the tests will pass it is not required

Reply:
Currently I am only adding unit tests which should pass at all times but I can foresee some experimental features that might be listed as optional.

@hbagdi @robscott Do you have any opinions on whether this new job should be marked optional now?

Reply:
> Do you have any opinions on whether this new job should be marked optional now?

I think we should start with `optional: false` since that's the whole purpose of adding these tests. We can keep the tests in a temporary dev branch if they happen to be temporarily breaking.